config/m4/ucm.m4

```diff
@@ -68,3 +68,5 @@ AC_CHECK_LIB([tcmalloc], [tc_malloc],
               TCMALLOC_LIB="-ltcmalloc"],
              [have_tcmalloc=no])
 AM_CONDITIONAL([HAVE_TCMALLOC],[test "x$have_tcmalloc" = "xyes"])
+
+AC_CHECK_FUNCS([__clear_cache], [], [], [])
```

Comment:
lets issue a warning if not present

Reply:
fail !

Reply:
internal implementation added, configure will prompt warning message